action.yml

```diff
@@ -37,33 +40,33 @@ runs:
       run: |
         set -ex
         for world in ${{ inputs.worlds }}; do
-          wit-bindgen markdown wit -w $world --check --html-in-md
+          wit-bindgen markdown ${{ inputs.directory }} -w $world --check --html-in-md
         done
 
     - name: Print generation instructions
       if: ${{ failure() }}
       shell: bash
       run: |
         echo 'Failed to verity that `*.md` files are up-to-date with'
-        echo 'their `wit/*.wit` counterparts. The `wit-bindgen` tool needs to'
+        echo 'their `${{ inputs.directory }}/*.wit` counterparts. The `wit-bindgen` tool needs to'
         echo 'be rerun on this branch and the changes should be committed'
         echo ''
-        echo '  cargo install wit-bindgen-cli@{{ inputs.wit-bindgen }}'
-        echo '  wit-bindgen markdown wit --html-in-md'
+        echo '  cargo install wit-bindgen-cli@${{ inputs.wit-bindgen }}'
```

Comment:
@sunfishcode: were you sure the `{{ ... }}` form was going to interpolate correctly here? I prepended `$` because I thought it wouldn't... but that's not really going to work either with how `'` works in bash. Not sure what to do.

Reply:
I think you're right. The `${{ ... }}` should get expanded by some templating before this gets intepreted by bash, so this probably works. I'm not 100% positive about this, but it's more likely to work now than before without the `$` in any case :upside_down_face: